streamr-docker-dev/bin.sh

```diff
@@ -26,7 +26,7 @@ SERVICE_COMBINATIONS=(
 "cassandra init_keyspace mysql redis smtp nginx broker-node-storage-1 broker-node-no-storage-1 broker-node-no-storage-2"
 "cassandra init_keyspace mysql redis smtp nginx tracker"
 "cassandra init_keyspace mysql redis smtp nginx "
-"cassandra init_keyspace mysql redis smtp nginx tracker tracker broker-node-storage-1 broker-node-no-storage-1 broker-node-no-storage-2 engine-and-editor"
+"cassandra init_keyspace mysql redis smtp nginx tracker broker-node-storage-1 broker-node-no-storage-1 broker-node-no-storage-2 engine-and-editor platform"
```

Comment:
How would the platform start everything but the platform?

Reply:
Hmm.. if I make a new target 6 with platform and have 5 without platform?

Reply:
Would that work?